_projects/add-e2ee-to-more-clients.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "Adding end-to-end encryption to more clients"
+desc: "All Matrix clients should support end-to-end encryption"
+difficulty: "Hard"
+issues:
+mentors:
+ - ara4n
+ - uhoreg
+initiatives:
+ - GSoC
+tags:
+ - Encryption
+ - Client
+---
+
+A lot of effort in Matrix is being put into End-to-End Encryption using the Olm & Megolm cryptographic ratchets (as assessed by NCC Group - see https://matrix.org/blog/2016/11/21/matrixs-olm-end-to-end-encryption-security-assessment-released-and-implemented-cross-platform-on-riot-at-last/ for details.
+
+However, the current implementation has only landed in matrix-js-sdk, matrix-ios-sdk and matrix-android-sdk. As a result, any client or bridge or bot which isn't built on one of those codebases is currently out of luck.
```

Comment:
There is also some form of python SDK support as of last year.

Reply:
True, but it literally hasn't landed. I'm wary of referring to it since it was a gsoc contribution that is slowly getting merged.